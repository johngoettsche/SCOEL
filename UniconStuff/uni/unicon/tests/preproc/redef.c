#define m 1 2 3
#define m 1	2     3
m

#define f(a,b) a      b
#define f(   a   ,   b   ) a b
f(1,2)

#begdef r(a)
a
#enddef
#begdef r(a)
a

#enddef
r("hi")
