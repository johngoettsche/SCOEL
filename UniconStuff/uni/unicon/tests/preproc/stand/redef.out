#line 2 "redef.c"

1 2 3



1      2


"hi"

